src/main/java/polyglot/Main.java

```diff
@@ -64,12 +64,31 @@ public static void main(String[] args) {
         ? configLoader.getNamedConfiguration(arguments.configName().get())
         : configLoader.getDefaultConfiguration();
     logger.info("Loaded configuration: " + config.getName());
-
-    logger.info("Loading proto file descriptors");
+    
     FileDescriptorSet fileDescriptorSet = getFileDescriptorSet(config.getProtoConfig());
+    
+    if (!arguments.command().isPresent()) {
+      runLegacyCommands(fileDescriptorSet, config, arguments);
+    
+    } else {
+      switch (arguments.command().get()) {
```

Comment:
Could we also have a "call" command for the previous calling behavior? Then, we could model "call" as simply being the default.